setup.cfg

```diff
@@ -1,10 +1,10 @@
 # Inside of setup.cfg
 [metadata]
-description-file = README.md
-license_file = LICENSE
+description_file = README.md
+license_files = LICENSE
 
 [bdist_wheel]
 universal = 1
 
 [egg_info]
-egg_base = ./build
```

Comment:
Are we sure it does not create issues? I kind of remember a similar issue coming up a while ago when discussing with Francois

Reply:
We fixed it in 269741e1552a29f20fde0c9c6ebd7a4eb97f7045.

Now the `build` folder is created before the build process begins, and all the build-related files and folders are placed into it.